src/main/java/roboy/dialog/DialogSystem.java

```diff
@@ -1,173 +1,148 @@
-		analyzers.add(new Preprocessor());
-		analyzers.add(new SimpleTokenizer());
-		analyzers.add(new OpenNLPPPOSTagger());
-		analyzers.add(new DictionaryBasedSentenceTypeDetector());
-		analyzers.add(new SentenceAnalyzer());
-		analyzers.add(new OpenNLPParser());
-		analyzers.add(new OntologyNERAnalyzer());
-		analyzers.add(new AnswerAnalyzer());
-        analyzers.add(new EmotionAnalyzer());
-        analyzers.add(new SemanticParserAnalyzer(Config.PARSER_PORT));
-        //if(!Config.NOROS) {
-        //    analyzers.add(new IntentAnalyzer(rosMainNode));
-        //}
-
-        if (!rosMainNode.STARTUP_SUCCESS && Config.SHUTDOWN_ON_ROS_FAILURE) {
-            throw new RuntimeException("DialogSystem shutdown caused by ROS main node initialization failure.");
-        }
-
-        System.out.println("DM initialized...");
-
-        while(true) {
-
-//            while (!Vision.getInstance().findFaces()) {
-//                emotion.act(new FaceAction("angry"));
-//            }
-//            emotion.act(new FaceAction("neutral"));
-//            while (!multiIn.listen().lists.containsKey(Linguistics.ROBOYDETECTED)) {
-//            }
-
-            Personality smallTalk = new SmallTalkPersonality(new Verbalizer(), rosMainNode);
-            Input raw;
-            Interpretation interpretation;
-            List<Action> actions = smallTalk.answer(new Interpretation(""));
-
-
-            while (actions.isEmpty() || !(actions.get(0) instanceof ShutDownAction)) {
-                multiOut.act(actions);
-                raw = multiIn.listen();
-                interpretation = new Interpretation(raw.sentence, raw.attributes); //TODO: Input devices should immediately produce Interpretation objects
-                for (Analyzer a : analyzers) {
-                    interpretation = a.analyze(interpretation);
-                }
-                actions = smallTalk.answer(interpretation);
-            }
-            List<Action> lastwords = ((ShutDownAction) actions.get(0)).getLastWords();
-            multiOut.act(lastwords);
-            actions.clear();
-        }
-	}
-}
+package roboy.dialog;
+
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import roboy.context.Context;
+import roboy.dialog.action.Action;
+import roboy.dialog.personality.StateBasedPersonality;
+import roboy.io.*;
+import roboy.linguistics.sentenceanalysis.*;
+import roboy.memory.Neo4jMemory;
+import roboy.memory.DummyMemory;
+import roboy.memory.Neo4jMemoryInterface;
+import roboy.memory.nodes.Interlocutor;
+import roboy.ros.RosMainNode;
+import roboy.talk.Verbalizer;
+import roboy.util.ConfigManager;
+import roboy.util.IO;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Temporary class to test new state based personality.
+ * Will be be extended and might replace the old DialogSystem in the future.
```

Comment:
I assume it already has ;) Maybe copy the introductory comment from the old class, as far as applicable?